policies/assembler.md

```diff
@@ -0,0 +1,19 @@
+Policy for Accepting Assembler Optimisations
+============================================
+
+New assembler optimisations for any algorithm having a C based implementation in
+any provider are always acceptable (subject to standard review procedures) for
+all platforms in the master branch.
```

Comment:
It's unclear what you mean with algorithm. The way things are implemented in assembler can be totally different than they are in C that I wouldn't call it the same algorithm. I don't see a need to restrict the assembler to do exactly the same as the C version. But maybe you just mean functionality and that all functionality should be available on all platforms?

It's also unclear what you mean with "all platforms in the master branch". We have various generic targets, and a new target can always be added in the same pull request.

And then there is the "in any provider". Does that include external providers?